rts/motoko-rts/src/text_iter.rs

```diff
@@ -53,7 +53,7 @@ const ITER_TODO_IDX: u32 = 2;
 /// Returns a new iterator for the text
 #[ic_mem_fn]
 pub unsafe fn text_iter<M: Memory>(mem: &mut M, text: Value) -> Value {
-    let iter = alloc_array(mem, 3);
+    let iter = alloc_array(mem, TAG_ARRAY_T, 3);
```

Comment:
Note to myself: This is the other iterator case of a non-standard array. To be checked that it is not reachable by graph copy...